src/components/Settings/MobileTab.react.js

```diff
@@ -34,6 +34,13 @@ const CountryCode = styled.div`
   align-items: center;
 `;
 
+const InputText = styled.div`
+  margin-right: 2rem;
+  @media (max-width: 410px) {
+    margin-right: 2px;
```

Comment:
Not required

Reply:
@fragm3 So should I do it to margin-right: 0px

Reply:
It's the same thing you are applying in media queries and normally. Why use media queries anyway if doing so

Reply:
There is a difference of `px` and `rem`